client/blocks/followers-count/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+import { getSelectedSite } from 'state/ui/selectors';
+
+class FollowersCount extends Component {
```

Comment:
Given that this component has no state or lifecycle handlers, it would be a good candidate to be a [stateless function component](https://facebook.github.io/react/docs/components-and-props.html#functional-and-class-components) which tend to be simpler to read and reason about.